src/check/parse/tokenize.zig

```diff
@@ -1016,23 +1022,21 @@ pub const Tokenizer = struct {
 
                 // Numbers starting with 0-9
                 '0'...'9' => {
-                    _ = self.cursor.chompNumber(b);
+                    const tag = self.cursor.chompNumber(b);
                     const len = self.cursor.pos - start;
-                    try self.output.pushToken(.Int, start, len);
+                    try self.output.pushToken(tag, start, len);
                 },
 
                 // Lowercase identifiers
                 'a'...'z' => {
-                    self.cursor.pos += 1;
```

Comment:
Hmm, why remove this? This is one less char we need to (re)process

Reply:
It was messing up keywords lookup. Instead of seeing `import`, it would see `mport`.

Reply:
Ohh; good catch. Thanks!